nestkernel/connection_manager.cpp

```diff
@@ -1163,13 +1157,7 @@ nest::ConnectionManager::get_connections(
 #ifdef _OPENMP
 #pragma omp critical( get_connections )
 #endif
-        {
-          while ( !conns_in_thread.empty() )
-          {
-            connectome.push_back( conns_in_thread.front() );
-            conns_in_thread.pop_front();
-          }
-        }
+        append_move_connections_( conns_in_thread, connectome );
```

Comment:
This is fine in principle, but the data flow is not clear to the reader. If the method returned the reference to the connectome, one could write
```C++
connectome = append_from_thread( connectome, conns_in_thread );
```
In my eyes that would be much easier to read.